hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -119,6 +119,102 @@
+                registers).
+             '''
+    },
+    {
+      name: "RV_DM.DBG.DATA_REGS",
+      descr: '''
+                 The DMI regs data0 and data1 are implemented.
+             '''
+    },
+    {
+      name: "RV_DM.DBG.DMCONTROL_REG",
+      descr: '''
+                 The DMI register dmcontrol is implemented. The following
+                 optional fields have limited function or are not implemented:
+                  - hartreset - Not implemented, writes ignored read zero
+                  - hasel - HART masks not supported, writes ignored, read zero
+                  - setresethaltreq - Not implemented, writes ignored read zero
+                  - clrresethaltreq - Not implemented, writes ignored read zero
+             '''
+    },
+    {
+      name: "RV_DM.DBG.DMSTATUS_REG",
+      descr: '''
+                 The DMI register dmstatus is implemented. As HART masks are not
+                 supported the various allX/anyX register pairs are identical.
+             '''
+    },
+    {
+      name: "RV_DM.DBG.HARTINFO_REG",
+      descr: '''
+                The DMI register hartinfo is implemented.
+             '''
+    },
+    {
+      name: "RV_DM.DBG.HALTSUM0_REG",
+      descr: '''
+                The DMI register haltsum0 is implemented.
+             '''
+    },
+    {
+      name: "RV_DM.DBG.ABSTRACT_COMMAND",
+      descr: '''
+                The DMI registers abstractcs/command/abstractauto are
+                implemented.
+
+                Only the Access Register Command is implemented.
+             '''
+    },
+    {
+      name: "RV_DM.DBG.PROGBUF",
+      descr: '''
+                The DMI implements the program buffer, with all registers
+                progbuf0 - progbuf15 usable
+             '''
+    },
+    {
+      name: "RV_DM.DBG.SBA",
+      descr: '''
+                The DMI implements system bus access. sbdata0/sbdata1 and
+                sbaddress0/sbaddress1 are implemented. However as the data and
+                address bus with is 32-bits the second register of each has no
+                functionality but can still be read and written.
+             '''
+    },
+    {
+      name: "RV_DM.DBG.GDB",
+      descr: '''
+                The debug module will work with GDB connected via OpenOCD over
+                JTAG. In particular the following features will be functional
+                from GDB:
+                  - Halting execution and resetting
+                  - Setting, hitting and deleting breakpoints using all
+                    available hardware breakpoints
+                  - Single stepping
+                  - Reading and writing all registers
+                  - Reading and writing CSRs
+                  - Reading and writing memory
+                  - Setting the PC to jump to some location
+                  - Executing code from GDB (using the call command)
+             '''
+    }
+  ]
```

Comment:
Should non-debug reset (NDM) be a feature?

Reply:
Good point, yes I think it should